pynicotine/gtkgui/ui/settings/uploads.ui

```diff
@@ -62,7 +62,7 @@
               <object class="GtkLabel">
                 <property name="height-request">24</property>
                 <property name="hexpand">True</property>
-                <property name="label" translatable="yes">Autoclear finished/cancelled uploads from transfer list</property>
+                <property name="label" translatable="yes">Auto-remove finished and cancelled uploads from list</property>
```

Comment:
"Auto-remove" -> "Automatically remove"

Avoid using hyphenated jargon because it might not translate well.